solutions/kattis-catandmice.mdx

```diff
@@ -0,0 +1,105 @@
+---
+id: kattis-catandmice
+source: Kattis
+title: Cat and Mice
+author: Kevin Sheng
+---
+
+## Explanation
```

Comment:
most of the other ones on the guide say Solution

Reply:
I think explanation is convention now (the ones that say solution were probably older) see [here](https://usaco.guide/general/adding-solution#steps)